Source/Core/VideoCommon/VertexLoaderARM64.cpp

```diff
@@ -288,7 +288,7 @@ void VertexLoaderARM64::ReadColor(u64 attribute, int format, s32 offset)
 			else if (offset & 3) // Not aligned - unscaled
 				LDUR(scratch2_reg, src_reg, offset);
```

Comment:
This destination register should be scratch3_reg like the rest of them.
That fixes some of the decoding issue, but not all of it.